routers/api/v1/repo/release_attachment.go

```diff
@@ -242,13 +244,14 @@ func EditReleaseAttachment(ctx *context.APIContext, form api.EditAttachmentOptio
 
 	// Check if release exists an load release
 	releaseID := ctx.ParamsInt64(":id")
-	attachID := ctx.ParamsInt64(":attachment")
+	attachID := ctx.ParamsInt64(":asset")
```

Comment:
I guess you should change:
```
	// - name: attachment_id
	//   in: path
	//   description: id of the attachment to edit
	//   type: integer
	//   format: int64
	//   required: true
```
into
```
	// - name: asset
	//   in: path
	//   description: id of the attachment to edit
	//   type: integer
	//   format: int64
	//   required: true
```

Reply:
Now that I think of it, why `asset`? It should be `attachment_id`, or else the swagger documentation needs update.

Reply:
Does swagger and macaron have to match? The GET operation already look like this (and work). There are other APIs where different variables are used in swagger and macaron.

Reply:
But sure, this difference is probably why the error was introduced.

Reply:
I guess you're right.